### DISCUSSION TITLE
Improove deploy logging and issues on deploy static files

### DISCUSSION THREAD

**Discussion by deniscass** [breezefront/community#335] [2. Ideas]
Hello guys, all right?
I recently went through a problem that took me a long time to find the solution. 
As usual, I implemented a child theme of the Breeze Blank Theme and made all the changes in development. When I went to deploy the theme in a production environment, the theme was not generated correctly with the command bin/magento setup:static:deploy -f pt_BR. It generated the files partially, missing some javascript and css. And there were a lot of JavaScript errors in the frontend. 
The strange thing is that the above command did not generate any error messages to make it easier to solve. 
Then I tried to check for possible failures, countless clean cache and nothing.
Then I was trying to remove old modules and themes that were still in production and I found out that another theme based on Luma was conflicting in production, The theme was HiddenTechnologiesBizquick free.

For some reason, the theme was conflicting, and when I went to generate my child theme Local/b2c-theme, which is a child of the Breeze Blank theme, it did not generate the static files correctly. I tried various alternatives such as putting Magento 2.4.6 p8 production into default mode and even development mode, checked if it was a "Content Security Policy" error, reviewed file and folder permissions, and nothing solved it. After removing the conflict and some third-party Magento 2 plugins, the error changed, and it stopped generating the theme deploy at all. An error message appeared stating that Breeze Blank could not find the parent theme during the deploy build. Finally, I identified that the theme table in the database was missing the parent fields for some reason, and that was the cause. I imagine it was some migration problem from the Magento 2.3 database. This account is to help anyone who encounters a similar problem in the new versions of Magento and also to suggest the possibility of improvement in logging behavior in deploy process. 

Thanks for this amazing and fast theme!

**Comment by vovayatsyuk** [comment]
Hello!

So the issue was the missing "parent" value in a `theme` DB table.

From my experience, a lot of issues are also caused by incorrect value in `type` column of the same table.

I'm not sure if it's a proper place to fix these issues in Breeze. All of the deployment is made by Magento core, since we fully rely on its functionality and we do not modify LESS processing.

Additionally, it's not clear to me on what should I do to deploy theme without errors in this case.
